mw-webapp/src/logic/wayPage/WayPage.tsx

```diff
@@ -340,6 +336,23 @@ export const WayPage = (props: WayPageProps) => {
         onChangeFinish={(text) => changeWayName(way, text)}
         isEditable={isOwner}
       />
+      {isOwner &&
+        <Button
+          value="Delete way"
+          onClick={() => renderModalContent({
```

Comment:
I don't like this line, but ok for now, please refactor as soon as possible

Reply:
Added comment